GameData/RealismOverhaul/Engine_Configs/AJ10_190_Config.cfg

```diff
@@ -30,9 +30,9 @@
 
     @MODULE[ModuleGimbal],*
     {
-        @gimbalRange = 6.0
-        %useGimbalResponseSpeed = True
-        %gimbalResponseSpeed = 16
+        gimbalRange = 6.0
```

Comment:
Why did you remove the MM modifiers? If an engine module already has them then the final output will include duplicates of the parameters.